app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/IndividualTriageAnswers.scala

```diff
@@ -21,12 +21,15 @@ import play.api.libs.json.{Json, OFormat}
 final case class IndividualTriageAnswers(
   individualUserType: Option[IndividualUserType],
   numberOfProperties: Option[NumberOfProperties],
+  disposalMethod: Option[DisposalMethod],
+  wasAUKResident: Option[Boolean],
+  wasResidentialProperty: Option[Boolean],
   disposalDate: Option[DisposalDate]
 )
 
 object IndividualTriageAnswers {
 
-  val empty: IndividualTriageAnswers = IndividualTriageAnswers(None, None, None)
+  val empty: IndividualTriageAnswers = IndividualTriageAnswers(None, None, None, None, None, None)
```

Comment:
So this will grow linearly with the number of triage questions? Isn't there 80+ questions which will break the case class limit for number of fields?

Reply:
For the triage questions there's going to be a limited number of questions so this shouldn't get massively big. Also, different states within the triage section may get their own case class depending on how it pans out. The whole return itself will have loads of questions in it, but it should be described using more than one case class

Reply:
OK, let's see how to unfolds...